problemsets-mat223/modules/module6-exercises.tex

```diff
@@ -1,17 +1,73 @@
 \begin{exercises}
+	% Topics
+	% formal + intuitive definition of subspace
+	% relationship between subspace and spans
+	% prove a set is a subspace
+	% basis + dimension of a subspace
 	\begin{problist}
-		\prob  Example problem with parts
+		\prob  State the definition of a subspace.
+
+		\prob  True of False:
+		\begin{enumerate}
+			\item All spans are subspaces.
+			\item All subspaces can be expressed as spans.
+			\item All translated spans are subspaces.
+			\item The empty set is a subspace.
+			\item The set $\Set*{\mat{1\\2}, \mat{2\\3}}$ is a subspace.
+		\end{enumerate}
+
+		\prob If the following is a subspace, prove it. Otherwise, explain why not.
 		\begin{enumerate}
-			\item   The first part
-			\item   The second part
+			\item Let $\mathcal T \subseteq \R^2$ be the complete solution to $3x-y=0$.
+			\item Let $\mathcal U \subseteq \R^2$ be the complete solution to $\frac{1}{2}x-6y=0$.
+			\item Let $\mathcal V \subseteq \R^2$ be the complete solution to $x-5y-1=0$.
+			\item Let $\mathcal W \subseteq \R^3$ be the complete solution to $x+2y-3z=0$.
+			\item Let $\mathcal X \subseteq \R^3$ be the complete solution to
+			$ex-\pi y + z\ln 2=0$.
+			\item Let $n \in \N$. Let $a_1,...,a_n \in \R$. Let $\mathcal Q \subseteq \R^n$ be the complete solution to $a_1x_1+a_2x_2+...+a_{n-1}x_{n-1}+a_nx_n=0$.
 		\end{enumerate}
-		\begin{solution}
-		Solutions to the parts
-			\begin{enumerate}
-				\item   $(2,4,8)$
-				\item   $(15,6,-21)$
-			\end{enumerate}
-		\end{solution}
 
+		\prob If the following is a subspace, prove it. Otherwise, explain why not.
+		\begin{enumerate}
+			\item Let $\mathcal B \subseteq \R^2$ be the the line in vector form
+			$\vec x = t\mat{5\\-7}+\mat{1\\2}$
+			\item Let $\mathcal A \subseteq \R^2$ be the the line in vector form
+			$\vec x = t\mat{-3\\4}$
+			\item Let $\mathcal C \subseteq \R^3$ be the the line in vector form
+			$\vec x = t\mat{1\\0\\5}$
+			\item Let $\mathcal D \subseteq \R^3$ be the the line in vector form
+			$\vec x = t\mat{2\\3\\4}+s\mat{10\\20\\131}+\mat{0\\0\\6}$
+			\item Let $\mathcal E \subseteq \R^3$ be the the line in vector form
+			$\vec x = t\mat{5\\7\\1}+s\mat{2\\-2\\1}$
+		\end{enumerate}
+
+		\prob Show that the following spans are subspaces.
```

Comment:
We have a theorem that says all spans are subspaces, right? Change this to "Use the definition of subspace to show each span below is a subspace"